tmt/steps/prepare/install.py

```diff
@@ -321,6 +335,11 @@ def install_debuginfo(self) -> None:
             self.guest.package_manager.check_presence(
                 *[Package(f'{package}-debuginfo') for package in self.debuginfo_packages])
 
+    def make_cache(self) -> None:
+        """ Generate metadata cache """
+
+        self.guest.package_manager.makecache()
```

Comment:
* can this be the default implementation of `make_cache()` method?
* `make_cache()` calling `makecache()` - please, pick one naming scheme :)
* hm, and does `make_cache()` even have to exist? It does not seem to bring added value, `install()` can call `self.guest.package_manager.makecache()` directly.

Reply:
Now it is calling it directly. I remember mypy complaining about not recognizing dynamically added methods that are defined only in specific subclasses. `makecache` was declared only in Dnf class in dnf.py, so I wanted to make sure it is called only by `make_cache` method of InstallDnf class from install.py and not by any other package manager, which would throw an error I believe. 

Now there is refresh_metadata method in every package manager class, so this is no longer necessary and removed. 

(make_cache x makecache - it felt wrong to me to have two methods named the same in two .py files, one calling the other. But my solution was probably even worse :-) )